asciidoctorj-arquillian-extension/build.gradle

```diff
@@ -0,0 +1,28 @@
+dependencies {
+  compile project(path: ':asciidoctorj')
+  compile project(path: ':asciidoctorj-test-support', configuration: 'tests')
+  compile "org.jboss.arquillian.container:arquillian-container-spi:$arquillianVersion"
+  compile "org.jboss.arquillian.container:arquillian-container-test-spi:$arquillianVersion"
+  compile "org.jboss.arquillian.container:arquillian-container-impl-base:$arquillianVersion"
+  compile "org.jboss.arquillian.container:arquillian-container-test-impl-base:$arquillianVersion"
+}
+
+jar.dependsOn project(':asciidoctorj').testClasses
+
+jar {
+  from project(':asciidoctorj').sourceSets.test.output
+  include 'org/asciidoctor/arquillian/**/*'
```

Comment:
Maybe this module could be skipped on Java 6 (like loading publish.gradle is skipped). 
In the end the only sense of the Arquillian extension module is to export these classes, so it should be no problem to publish with Java >= 7. 



> Am 06.06.2015 um 17:50 schrieb Frank Becker <notifications@github.com>:
> 
> In asciidoctorj-arquillian-extension/build.gradle:
> 
> > @@ -0,0 +1,28 @@
> > +dependencies {
> > +  compile project(path: ':asciidoctorj')
> > +  compile project(path: ':asciidoctorj-test-support', configuration: 'tests')
> > +  compile "org.jboss.arquillian.container:arquillian-container-spi:$arquillianVersion"
> > +  compile "org.jboss.arquillian.container:arquillian-container-test-spi:$arquillianVersion"
> > +  compile "org.jboss.arquillian.container:arquillian-container-impl-base:$arquillianVersion"
> > +  compile "org.jboss.arquillian.container:arquillian-container-test-impl-base:$arquillianVersion"
> > +}
> > +
> > +jar.dependsOn project(':asciidoctorj').testClasses
> > +
> > +jar {
> > +  from project(':asciidoctorj').sourceSets.test.output
> > +  include 'org/asciidoctor/arquillian/**/*'
> Ok. So the extra-configurations plugin is incompatible to java6, like the publishing plugin.
> What about that issue?
> 
> —
> Reply to this email directly or view it on GitHub.
>